content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+
+## How long and complex will it be?
+
+It depends on the contribution. Some contributions may be spot on and can be reviewed in a few minutes. Other contributions may require a more detailed review and changes to be made and can take longer.
+However we approximate it to take about 3-15 mins for one to review a document, and getting better with time.
+
+## Where to start
+
+To get started, we will need to understand a number things. The nature of the contributions you will be reviewing, where to get the contributions to review and the tools that will help you in reviewing the contributions.
+
+### The nature of the contributions
+
+The contributions you will be reviewing are contractual documents of digital services. These are documents that govern the relationship between two or more parties.
+They are not the original documents, but rather accurate tracking and data about these documents. If the documents are represented accurately, it will be easier to follow on with any subsequent changes in the document. Contributors track these documents (sometimes, anonymously) by submitting a pull request either using the **GUI contributing tool** (A tool that helps contributors add documents to the project), or creating a JSON file and adding it via a pull request. You can find more information about pull requests [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+There are three types of contributions you'll come across:
+
+- Adding contributions to declarations
+- Updating contributions to declarations
+- Removal contributions to declarations
+
+The method used to review each of these types of contribution varies, and you'll find a detailed explanation of this as you proceed with this document.
+
+### Where to find the contributions
+
+These contributions are housed in the [`contrib-declarations`](https://github.com/OpenTermsArchive/contrib-declarations) repository under the pull requests tab. You can find the pull requests [here](https://github.com/OpenTermsArchive/contrib-declarations/pulls).
+
+## How To Review Pull Requests that Add Declarations
+
+When a contribution is made using the contribution tool, the OTA-bot will create a pull request. This pull request typically consists of several checks, which guides the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
+You should focus should be on two things: accuracy and quality.
+
+- Accuracy is about making sure that the contribution document is accurate and tracks **significant** sections that actually make a legal impact when changed.
+- Quality is about making sure that the contribution is of high quality and uses stable CSS selectors that will make sure the document will need less maintainace over time.
+
+### Step-by-step Review Guide
+
+1. Click on the inspect the declaration suggestion link to view contribution using the contribution tool.
```

Comment:
I try to avoid naming the tool itself and buttons as they might change name over time 🙂 
```suggestion
1. Click on the _Inspect the declaration_ link to view the declaration in a graphical user interface.
```